### PR TITLE
Improve formatting of copyright headers

### PR DESCRIPTION
**Pull request by dalinaum**
* Confidential -> Apache
* Add missing copyright headers.
* Add new lines after headers.
* Delete unnecessary new lines in headers

Some of them were created on 2015. (2014 -> 2015)

### CODE REVIEW

**Comment by dalinaum** [comment]
It's a small patch. Please review this @realm/java

File: realm/realm-library/src/androidTest/java/io/realm/internal/test/ColumnTypeData.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2014 Realm Inc.
```
Comment (emanuelez) [inline realm/realm-library/src/androidTest/java/io/realm/internal/test/ColumnTypeData.java]: We shall not change the year if the file has been created in 2014

Comment (dalinaum) [inline realm/realm-library/src/androidTest/java/io/realm/internal/test/ColumnTypeData.java]: Has files of internal test been created in 2014? When I used `git log`, there were no history before 2015. If they had been imported from other directory, I should modify them all as 2014.

Comment (dalinaum) [inline realm/realm-library/src/androidTest/java/io/realm/internal/test/ColumnTypeData.java]: It seems my mistake. I fixed them.

**Comment by cmelchior** [comment]
Nice cleanup :+1:

**Comment by beeender** [comment]
:+1:

**Pull request merged**